VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/participant-panel.model.spec.ts

```diff
@@ -60,4 +61,19 @@ describe('ParticipantPanelModel', () => {
         model = mapper.mapFromParticipantUserResponse(participant);
         expect(model.isJudicialOfficeHolder).toBeFalsy();
     });
+
+    it('should return false when participant is a quick link observer and status is in hearing', () => {
+        participant.role = Role.QuickLinkObserver;
+        participant.status = ParticipantStatus.InHearing;
+        participant.hearing_role = HearingRole.QUICK_LINK_OBSERVER;
+        model = mapper.mapFromParticipantUserResponse(participant);
+        expect(model.isWitnessObserverReadyToJoin).toBeFalsy();
+    });
+
+    it('should return false when participant is a witness and status is in hearing', () => {
+        participant.hearing_role = HearingRole.WITNESS;
+        participant.status = ParticipantStatus.InHearing;
+        model = mapper.mapFromParticipantUserResponse(participant);
+        expect(model.isWitnessObserverReadyToJoin).toBeFalsy();
+    });
```

Comment:
Could we get true versions  for these when not in hearing?

Reply:
Done